e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/helpers/index.ts

```diff
@@ -71,24 +71,27 @@ export const navigateToEntitySelectionStep = (
 
   cy.log("select an experience");
 
-  if (experience === "chart") {
-    cy.findByText("Chart").click();
-  } else if (experience === "exploration") {
-    cy.findByText("Exploration").click();
-  } else if (experience === "browser") {
-    cy.findByText("Browser").click();
+  const hasEntitySelection =
+    experience !== "exploration" && experience !== "metabot";
+
+  const labelByExperience = match(experience)
+    .with("chart", () => "Chart")
+    .with("exploration", () => "Exploration")
+    .with("browser", () => "Browser")
+    .with("metabot", () => "Metabot")
+    .otherwise(() => undefined);
+
+  if (labelByExperience) {
+    cy.findByText(labelByExperience).click();
   }
 
   // exploration template does not have the entity selection step
```

Comment:
```suggestion
  // exploration template, and Metabot component do not have the entity selection step
```

Reply:
Fixed in https://github.com/metabase/metabase/pull/64388/commits/aceca54f38e71ae7144fb52f5e69ba31005ae60b, thanks!